docs/Release_Guide/index.rst

```diff
@@ -2,7 +2,57 @@
 Release Guide
 =============
 
-This METplus Release Guide provides detailed instructions for creating software releases from the METplus component repositories.
+This METplus Release Guide provides detailed instructions for METplus
+developers for creating software releases for the METplus component
+repositories.  **This Release Guide is intended for developers creating
+releases and is not intended for users of the software.**
+
+.. _releaseCycleStages:
+
+Stages of the METplus Release Cycle
+===================================
+
+Development Release
+-------------------
+
+Beta
+^^^^
+
+Beta releases are a pre-release of the software to give a larger group of
+users the opportunity to test the recently incorporated new features,
+enhancements, and bug fixes.  Beta releases allow for continued
+development and bug fixes before an official release.  There are many
+possible configurations of hardware and software that exist and installation
+of beta releases allow for testing of potential conflicts.
+
+Release Candidate (rc)
+^^^^^^^^^^^^^^^^^^^^^^
+
+A release candidate is a version of the software that is nearly ready for
+official release but may still have a few bugs.  At this stage, all product
+features have been designed, coded, and tested through one or more beta
+cycles with no known bugs.  It is code complete, meaning that no entirely
+new source code will be added to this release.  There may still be source
+code changes to fix bugs, changes to documentation, and changes to test
+cases or utilities.
+
+Official Release
+----------------
+
+An official release is a stable release and is basically the release
+candidate, which has passed all tests.  It is the version of the code that
+has been tested as thoroughly as possible and is reliable enough to be
+used in production.
+
+Bugfix Release
+--------------
+
+A bugfix release introduces no new features, but fixes bugs in previous
+official releases and targets the most critical bugs affecting users. 
+
+Instructions Summary
+====================
+
 Instructions are provided for three types of software releases:
 
 #. **Official Release** (e.g. vX.Y.Z) from the develop branch
```

Comment:
Official Release (e.g. vX.Y.Z) from the develop branch (becomes the new main_vX.Y branch)

Reply:
Thanks @ksearight.  I made the suggested change.